components/search/DateFilter.stories.js

```diff
@@ -0,0 +1,38 @@
+import { storiesOf } from '@storybook/vue';
+import DateFilter from './DateFilter.vue';
+
+const i18n = {
+  locale: 'en',
+  messages: {
+    en: {
+      dateFilter: {
+        legend: 'Date range',
+        startDate: 'Start date',
+        endDate: 'End date'
+      }
+    }
+  }
+};
+
+storiesOf('Search', module)
+  .add('Date Filter', () => ({
+    i18n,
+    components: { DateFilter },
+    data() {
```

Comment:
Is this necessary? Shouldn't the data function in the component handle this?

Reply:
This is fake data pre-populate the form